.vscode/settings.json

```diff
@@ -2,11 +2,11 @@
 	"[rust]": {
 		"editor.formatOnSave": true,
 		"editor.formatOnPaste": true,
-		"editor.defaultFormatter": "rust-lang.rust-analyzer",
+		"editor.defaultFormatter": "matklad.rust-analyzer",
```

Comment:
[Rust Analyser is now part of the rust-lang org](https://blog.rust-lang.org/2022/02/21/rust-analyzer-joins-rust-org.html). If you are still using the old extension you should consider updating.

Reply:
Oh, thanks for letting me know. I didn't mean to change that...